README.md

```diff
@@ -20,6 +20,12 @@ If you're feeling adventurous, you can also install the from the master branch:
 ```
 $ npm install -g git+https://git@github.com/ConsenSys/truffle-security.git
 ```
+### Windows only
+Before installing plugin make sure to install `windows-build-tools`
```

Comment:
It might be nice to mention that this requires an elevated account. We can also link to `node-gyp`'s instructions for this, because that is the dependency that needs it. https://github.com/nodejs/node-gyp#option-1